pkg/kubelet/remote/remote_runtime_test.go

```diff
@@ -45,7 +45,7 @@ func createAndStartFakeRemoteRuntime(t *testing.T) (*fakeremote.RemoteRuntime, s
 }
 
 func createRemoteRuntimeService(endpoint string, t *testing.T) internalapi.RuntimeService {
-	runtimeService, err := NewRemoteRuntimeService(endpoint, defaultConnectionTimeout)
+	runtimeService, err := NewRemoteRuntimeService(endpoint, defaultConnectionTimeout, "k8s.gcr.io/pause:3.1")
```

Comment:
we have the parent container image:tag defined in too many locations, e.g.
https://github.com/kubernetes/kubernetes/blob/d24fe8a801748953a5c34fd34faa8005c6ad1770/pkg/kubelet/dockershim/docker_sandbox.go#L42
https://github.com/kubernetes/kubernetes/blob/446cf20c9f5dfbfef3aa61d53e91c76dea340030/cmd/kubelet/app/options/container_runtime.go#L30

i think it needs to find a home possibly under pkg/kubelet/ as a constant.
[1]

Reply:
Yes, I agree. I could make the `defaultPodSandboxImageName` and `defaultPodSandboxImageVersion` public in `cmd/kubelet/app/options/container_runtime.go`, WDYT?

Reply:
ideally it should be under pkg/kubelet, because the cmd/kubelet package should not be included elsewhere (although we do have similar instances in the codebase).

Reply:
Alright, I will go for adding it to `pkg/kubelet/config/defaults.go`